src/shared/components/results/ResultsButtons.tsx

```diff
@@ -132,9 +142,9 @@ const ResultsButtons: FC<ResultsButtonsProps> = ({
             })}
           />
         </Button>
-        {viewMode === ViewMode.TABLE && namespace !== Namespace.idmapping && (
-          <CustomiseButton />
-        )}
+        {!notCustomisable &&
+          viewMode === ViewMode.TABLE &&
+          namespace !== Namespace.idmapping && <CustomiseButton />}
```

Comment:
I think we should be consistent and either use `!notCustomisable` everywhere (id mapping included, which would involve passing the prop through `ResultsDataHeader`, or only by using the namespace (although this might be more tricky?)

Reply:
Changed to use notCustomisable everywhere needed.
In the same commit, I fixed an issue with the ID mapping download not working for "all"